tests/wpunit/AccessFunctionsTest.php

```diff
@@ -297,23 +272,32 @@ public function testFilterInputFields() {
 			return $fields;
 		}, 10, 4 );
 
-		$actual = graphql([ 'query' => $query ]);
-
-		codecept_debug( $actual );
+		$response = $this->graphql( compact( 'query' ) );
 
 		/**
 		 * Map the names of the inputFields to be an array so we can properly
 		 * assert that the input field is there
 		 */
 		$field_names = array_map( function( $field ) {
 			return $field['name'];
-		}, $actual['data']['__type']['inputFields'] );
+		}, $response['data']['__type']['inputFields'] );
 
 		codecept_debug( $field_names );
 
-		$this->assertArrayNotHasKey( 'errors', $actual );
+		$this->assertArrayNotHasKey( 'errors', $response );
 		$this->assertContains( 'testInputField', $field_names );
 
 	}
 
+	public function testReplaceGraphQLFieldName() {
+		replace_graphql_field_name( 'RootQuery', 'user', 'wpUser' );
```

Comment:
Ok, so I think we should assert that `user` exists, and `wpUser` does not. Then use the `replace_graphql_field_name()` then assert that `wpUser` does exist and `user` does not.